lib/osf-components/addon/components/new-project-modal/template.hbs

```diff
@@ -14,29 +14,45 @@
                             class='form-control project-name'
                             required=true
                             value=nodeTitle
-                            type="text"
-                            enter=(action create nodeTitle description searchSelected.id)
+                            type='text'
+                            enter=(action 'create')
                         }}
                     </label>
                 </div>
 
                 {{#if institutions.length}}
                     <div class="form-group m-v-sm" local-class="NewProject__institution-selector">
                         <label class="f-w-lg text-bigger">{{t 'new_project.affiliation'}}</label>
-                        <a role="button" onclick={{action removeAll}} {{action 'click' 'button' 'Dashboard - New Project - remove_all' target=analytics}}>{{t 'new_project.remove_all'}}</a>
-                        <a role="button" onclick={{action selectAll}} {{action 'click' 'button' 'Dashboard - New Project - select_all' target=analytics}}>{{t 'new_project.select_all'}}</a>
+                        <a
+                            role="button"
+                            onclick={{action 'removeAllInstitutions'}}
+                            {{action 'click' 'button' 'Dashboard - New Project - remove_all' target=analytics}}
+                        >
+                            {{t 'new_project.remove_all'}}
+                        </a>
+                        <a
+                            role="button"
+                            onclick={{action 'selectAllInstitutions'}}
+                            {{action 'click' 'button' 'Dashboard - New Project - select_all' target=analytics}}
+                        >
+                            {{t 'new_project.select_all'}}
+                        </a>
                         <table>
                             <tr>
                                 {{#each institutions as | institution |}}
                                     <td>
-                                        <a role='button' onclick={{action selectInstitution institution}} {{action 'click' 'button' 'Dashboard - New Project - select_institution' target=analytics}}>
+                                        <a
+                                            role='button'
+                                            onclick={{action 'selectInstitution' institution}}
+                                            {{action 'click' 'button' 'Dashboard - New Project - select_institution' target=analytics}}
```

Comment:
Why not all of them?

Reply:
I moved all except the "Cancel" button, which doesn't have its own action, and the "Go to new project" link.